.env.example

```diff
@@ -17,8 +17,10 @@ DATABASE_URL="postgres://opencap:password@127.0.0.1:54321/opencap"
 # You can generate a new secret on the command line with:
 # openssl rand -base64 32
 # https://next-auth.js.org/configuration/options#secret
-# NEXTAUTH_SECRET=""
+# NEXTAUTH_SECRET="H3oBgcoF8//UgPAsGWMu8flAvX42s0qCVZg08zkiSNU="
 NEXTAUTH_URL="http://localhost:3000"
+GOOGLE_CLIENT_ID="480430891015-m7kiprj3meqk90j5erdv68133q6id20v.apps.googleusercontent.com"
+GOOGLE_CLIENT_SECRET="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"
```

Comment:
Okay. Thanks for the correction. I added them because these will not be the actual keys that will be used to get the oAuth running. We'd have to switch the keys to the ones you'll provide
How would you like me to share the keys for others to test if the oAuth works?